src/frontend/components/Home.tsx

```diff
@@ -1,256 +1,193 @@
 import * as React from 'react';
```

Comment:
Mainly designing the layout prior to moving to components. Still need to work on incorporating tailwindcss into some css-in-js solution.